src/journal/journald.c

```diff
@@ -52,7 +52,7 @@ int main(int argc, char *argv[]) {
                 goto finish;
 
         server_vacuum(&server, false);
-        server_flush_to_var(&server);
+        server_flush_to_var(&server, true);
```

Comment:
Well, there is a race anyway. I created `/var/log/journal` (I didn't use that hack: https://github.com/systemd/systemd/commit/c214ee6f15fd92342a7526fd99dc768d56950ac7 and reverted another hack https://github.com/systemd/systemd/pull/2057/commits/6b95e47067e52be60b37307d57f1ed2b5419b9d5). I got:
```
$ ls -ld /var/log/journal/e481b78163a34d7ab77c076b2aba27d8/system.journal
-rw-r-----+ 1 root root 4845568 Dec 15 22:48 /var/log/journal/e481b78163a34d7ab77c076b2aba27d8/system.journal
```
So, I'm not sure what does this commit fix